pkg/target/templating_test.go

```diff
@@ -0,0 +1,172 @@
+/**
+ * Copyright (c) 2020-present Snowplow Analytics Ltd.
+ * All rights reserved.
+ *
+ * This software is made available by Snowplow Analytics, Ltd.,
+ * under the terms of the Snowplow Limited Use License Agreement, Version 1.0
+ * located at https://docs.snowplow.io/limited-use-license-1.0
+ * BY INSTALLING, DOWNLOADING, ACCESSING, USING OR DISTRIBUTING ANY PORTION
+ * OF THE SOFTWARE, YOU AGREE TO THE TERMS OF SUCH LICENSE AGREEMENT.
+ */
+
+package target
+
+import (
+	"testing"
+
+	"github.com/snowplow/snowbridge/pkg/models"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTemplating_WithPrettyPrint(t *testing.T) {
+	assert := assert.New(t)
+
+	rawTemplate :=
```

Comment:
Oh we define it here! We can just make this a global var in target, reuse it & remove the file, unless I've missed why the file exists?

Reply:
So here in `templating_test.go` I pass template as string directly to templating function.

Test in `http_test.go` is slightly different, because I test there whole `Write` function, including target creation. So it also checks if we read template from given file properly.

I followed the pattern of using `integration` dir from this TLS [test](https://github.com/snowplow/snowbridge/blob/71e306c65588af784686cb41205a8fa5550554c7/pkg/target/http_test.go#L633). I could also simply define this template file in current test directory, next to test file?

Reply:
Ah I see where you're coming from. Those TLS files are necessary for the server to access too, which is why they're housed there in the first place.

Seems like it's not worth the scope creep so I should accept the need for a file. If it must exist, the integration dir is as good a place as any for it then.

Just another thing on the list of QOL improvements across the project that we should hopefully make cleaner!